pkg/cli/alpha/config-gen/examples/kustomize/README.md

```diff
@@ -19,7 +19,7 @@ generated are provided as `resources` input, the inputs will be modified by the
 Install the latest version of `kustomize`.
 
 ```sh
-GO111MODULE=on go get sigs.k8s.io/kustomize/kustomize/v4
```

Comment:
cannot be go install?

Reply:
No, explanation in PR description.

Reply:
Hi @ryantking please see: https://github.com/kubernetes-sigs/kubebuilder/issues/2480